src/App.tsx

```diff
@@ -17,17 +17,15 @@ import './App.css';
 
 const App: FC = () => {
   const dispatch = useDispatch();
-  const signedIn = useSelector(authSelectors.selectAuthSignedIn);
-  const [routes, setRoutes] = useState(ROUTES.defaultRoutes);
+  const signedIn = useSelector(selectAuthSignedIn);
+  const [routes, setRoutes] = useState(defaultRoutes);
 
   useEffect(() => {
-    dispatch(authActions.checkAuth());
+    dispatch(checkAuth());
   }, [dispatch]);
```

Comment:
i don't think dispatch will be changed, hence its as good as watching `[]`. Not sure tho, some unit test could verify this.

Reply:
Yeah, I agree with you, react-redux already handles this but my linting is forcing it to be put in. Kinda weird